bin/dump_kubernetes.sh

```diff
@@ -123,6 +123,32 @@ dump_resources() {
       -o yaml > "${RESOURCES_FILE}"
 }
 
+dump_pilot_url(){
+  pilot_pod=$1
```

Comment:
These should all have local in front of them.
https://google.github.io/styleguide/shell.xml?showone=Use_Local_Variables#Use_Local_Variables